docs/presto-admin-commands.rst

```diff
@@ -2,6 +2,92 @@
 Presto-Admin Commands
 =====================
 
+.. _catalog-add:
+
+***********
+catalog add
+***********
+::
+
+    presto-admin catalog add [<name>]
+
+This command is used to deploy catalog configurations to the Presto cluster.
+`Catalog configurations <https://prestodb.io/docs/current/connector.html>`_ are
+kept in the configuration directory ``~/.prestoadmin/catalog``
+
+To add a catalog using ``presto-admin``, first create a configuration file in
+``~/.prestoadmin/catalog``. The file should be named ``<name>.properties`` and
+contain the configuration for that catalog.
+
+Use the optional ``name`` argument to add a particular catalog to your
+cluster. To deploy all catalogs in the catalog configuration directory,
+leave the name argument out.
+
+In order to query using the newly added catalog, you need to restart the
+Presto server (see `server restart`_): ::
+
+    presto-admin server restart
+
+Example
+-------
+To add a jmx catalog, create a file
```

Comment:
should we call jmx, tpch etc as connectors instead? That is how presto calls it, right? For eg: below(line 48) for Custom , we call it a connector. Also applies to the rest of this PR where we refer to tpch/jmx etc.